packages/vats/src/core/startWalletFactory.js

```diff
@@ -184,7 +184,11 @@ export const startWalletFactory = async (
 
   const marshaller = await E(board).getPublishingMarshaller();
   const poolBank = E(bankManager).getBankForAddress(poolAddr);
-  const ppFacets = await E(startGovernedUpgradable)({
+
+  // We cannot await the start of the provisionPool, since
```

Comment:
We did `await` when we rebooted the VM in the vaults upgrade, right? Any idea what changed?

https://github.com/Agoric/agoric-sdk/blob/ba3b776444425bbcb275ce918794d9259f8e15da/packages/vats/src/core/startWalletFactory.js#L169

Reply:
The `decentral-devnet-config.json` introduced coreProposal steps, which will be useful for clearer separation of bootstrap dependencies.  That clearer separation revealed the fact that `startWalletFactory` would not resolve until `inter-protocol` established the governance committee for the `provisionPool`.  See my updates to the PR description above.

I don't know what completed the cycle from the inter-protocol side of things back to the wallet factory.

Since this runs at chain bootstrap time, all steps will complete before the first block commits, so the provisionPool-less walletFactory isn't exposed to any externally-submitted transactions.

Reply:
ok